hax/treemakers/trigger.py

```diff
@@ -191,6 +191,9 @@ def get_data(self, dataset, event_list=None):
         # Using load_single_dataset instead of load will ensure no blindding cut is applies
         data, _ = hax.minitrees.load_single_dataset(dataset, ['Fundamentals', 'LargestPeakProperties'])
 
+        if data.empty:
+            return pd.DataFrame({})
```

Comment:
Perhaps you need to add event_number and run_number columns to the empty dataframe, otherwise the merging might fail later on. See https://github.com/XENON1T/hax/blob/45ab41a0ec269b6b9f3c3342f99dc49034bb8511/hax/minitrees.py#L381